src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeDecisionDateChecker.java

```diff
@@ -131,6 +130,7 @@ private LocalDate handleOutOfCountryAppeal(AsylumCase asylumCase, OutOfCountryDe
                         .orElseThrow(() -> new RequiredFieldMissingException("dateClientLeaveUk is not present")));
 
             case REFUSAL_OF_HUMAN_RIGHTS:
+            case REFUSE_PERMIT:
```

Comment:
Just want to check that the issue was only affecting this REFUSE_PERMIT scenario and not the others

Reply:
Verified that it was only affecting this option.